app/src/org/commcare/network/DebugDataPullResponseFactory.java

```diff
@@ -1,8 +1,5 @@
-package org.commcare.tasks.network;
+package org.commcare.network;
 
-import org.commcare.network.DataPullRequester;
-import org.commcare.network.HttpRequestGenerator;
-import org.commcare.network.RemoteDataPullResponse;
 import org.commcare.tasks.DataPullTask;
 
 import java.io.IOException;
```

Comment:
Now that this, and `DebugDataPullResponse` are being used in live code it would be good to remove mention of debugging in the name?